sdk/cosmos/azure-cosmos/test/test_config.py

```diff
@@ -39,7 +39,7 @@ class _test_config(object):
 
     #[SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="Cosmos DB Emulator Key")]
     masterKey = os.getenv('ACCOUNT_KEY', 'C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==')
-    host = os.getenv('ACCOUNT_HOST', 'https://localhost:443/')
+    host = os.getenv('ACCOUNT_HOST', 'https://localhost:8081/')
```

Comment:
The internal emulator uses 443. This change runs the tests against the public emulator on 8081